rclpy/rclpy/time_source.py

```diff
@@ -88,3 +108,14 @@ def clock_callback(self, msg):
         self._last_time_set = time_from_msg
         for clock in self._associated_clocks:
             clock.set_ros_time_override(time_from_msg)
+
+    def on_parameter_event(self, parameter_list):
```

Comment:
This method doesn't need to be public and can be hidden by adding a `_` prefix.

Reply:
Fixed